libcodechecker/server/config_db_model.py

```diff
@@ -0,0 +1,93 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+"""
+SQLAlchemy ORM model for the product configuration database.
+"""
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from sqlalchemy import *
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import *
+
+CC_META = MetaData(naming_convention={
```

Comment:
You **can not** move `Base` to a common place, at it is the base class of the ORM hierarchy. And the two hierarchies are separate, they cannot be joined by the same base class. It would result in the database creation always creating **both** schemas in every database, which eventually prohibits every chance of migration.